### ISSUE TITLE
Can't edit files on network drives

### ISSUE THREAD

**Issue by jerone** [sindresorhus/atom-editorconfig#28]
I'm having problems with text files opening on network drives. I can open folders and see all files in the treeview, but as soon try to open a file in the editor, nothing happens (no error, no message, no contents). This does not happen to images (`.png` & `.gif`). I originally reported it here: https://github.com/atom/atom/issues/2313#issuecomment-70653160, but after inspection (running `--safe`-mode and enabling/disabling all packages) this seems only related to atom-editorconfig.

Windows 8.1
Atom 0.174.0
Editorconfig 0.3.0

**Comment by okoetter** [comment]
I have the same problem. Under Windows, I can open files from mapped network drives but not from UNC paths. After uninstalling this addon, it worked fine.

**Comment by sindresorhus** [comment]
This plugin doesn't do anything with the path directly the issues is with https://github.com/editorconfig/editorconfig-core-js

Can you console.log the file path passed to this plugin: https://github.com/sindresorhus/atom-editorconfig/blob/d4ab369634e07d67dea7a834473b7af8189e6163/index.js#L13 and share it here?

I've opened an issue on the EditorConfig issue tracker: https://github.com/editorconfig/editorconfig/issues/195

**Comment by jerone** [comment]
After a lot of `console.log`'s I've found out that the code bugs on https://github.com/editorconfig/editorconfig-core-js/blob/master/editorconfig.js#L69 where `path.resolve(filepath).match(/^[^\\]+\\/)` resulting in `null`, making `[0]` not possible.

**Comment by jerone** [comment]
The real path is private, but it's something like: 
```
\\12.34.5.678\folder\folder\www.site.com_1.2.3a\folder\folder
```

**Comment by awayken** [comment]
:frowning: I'm having this issue, too.

**Comment by treyhunner** [comment]
@sindresorhus I just released version 0.12.2 of the JS core library which should fix this.

**Comment by awayken** [comment]
You guys are all awesome. Works like a charm! :clap:
